develop/tutorials/articles/260-forms/01-form-field-types/00-intro.markdown

```diff
@@ -37,10 +37,8 @@ Before getting started, learn the structure of a form field type.
 
 ## Anatomy of a Field Type Module [](id=anatomy-of-a-field-type-module)
 
-The `dynamic-data-mapping-type-*` modules in @product@'s source code are good
-templates to follow when developing your own field types. For example, look at
-the directory structure of the `dynamic-data-mapping-type-paragraph` module
-(version 3.0.0):
+All form field type modules have a similar structure. Here's the directory
+structure of the `dynamic-data-mapping-type-time` module (version 3.0.0):
```

Comment:
The version 3.0.0 is referring the paragraph field, so we need to change it to 1.0.0 or remove it. Any of the options are good.